Claudinee11/My-Dairy
# Ft sign up 168192912

What does this PR do?
- edit sign up page form

Description of Task to be completed
- A user should be able to use a form and on clicking the submit button, they are redirected to sign up page.
How should this be manually tested?
- clone the repo and switch to the branch ft-sign-up-168192912
- enter details in the field and click submit
- you should be successfully redirected to reset-sign up.html
What are the relevant pivotal tracker stories?
- 168192912

Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/50420731/64082436-d598ba00-cd73-11e9-9443-33d5e17fd484.png)
 Question:
